cmd/src/update_elevated_linux.go

```diff
@@ -0,0 +1,16 @@
+// +build !windows
+
+package main
+
+import (
+	"os"
+	"os/exec"
+)
+
+func runWithElevatedPrivilege() error {
+	cmd := exec.Command("sudo", "/bin/sh", "-c", "./src update")
```

Comment:
I don't think this will work, since `src` might not be in the current directory (`./`).

```suggestion
	cmd := exec.Command("sudo", "/bin/sh", "-c", "./src update")
```